.github/workflows/lint-pr.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
     name: Labeling for releasing
     runs-on: ubuntu-latest
     steps:
-      - uses: github/issue-labeler@v2.5 #May not be the latest version
+      - uses: github/issue-labeler@v3.1 #May not be the latest version
```

Comment:
See the reason at https://github.com/github/issue-labeler/pull/59#discussion_r1093345415.